docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -82,214 +68,347 @@ To enable your application to use the image util functionality:
 
    For more information on the YUV color space, see [http://www.fourcc.org/yuv.php](http://www.fourcc.org/yuv.php).
 
-4. To support the `image_util_transform_run()` function, which is used for all image transformations, set the source image and create a handle for it (to be used as the second parameter):
+4. To support `image_util_transform_run()`, which is used for all image transformations, set the source image and create a handle for it (to be used as the second parameter):
 
    ```
    ret = image_util_decode_create(&decode_h);
    ret = image_util_decode_set_input_path(decode_h, SAMPLE_FILENAME);
    ret = image_util_decode_set_colorspace(decode_h, colorspace);
-   ret = image_util_decode_set_output_buffer(decode_h, &img_source);
 
-   ret = image_util_decode_run(decode_h, &width, &height, &size_decode);
+   ret = image_util_decode_run2(decode_h, &src_image);
    ret = image_util_decode_destroy(decode_h);
+   ```
 
-   ret = media_format_create(&fmt);
-   ret = media_format_set_video_mime(fmt, colorspace);
-   ret = media_format_set_video_width(fmt, width);
-   ret = media_format_set_video_height(fmt, height);
+<a name="image_colorspace"></a>
+## Converting Image Color Space
 
-   ret = media_packet_create_alloc(fmt, NULL, NULL, &src);
-   ret = media_packet_get_buffer_size(src, &size);
+To convert an image from one color space of an image to another:
```

Comment:
Hello. Thanks for your review.
I agree the duplicated word and the meaning of sentence was wrong.
Because the feature is converting the colorspace of an image, I think that the sentence should be below. @all, Could you please check?

To convert one color space of an image to another:

Reply:
@surbhig12 Would please check the suggestions? I think both from @jiyong2 are good.

In my opinion, 
If we are going to use "To convert one color space of a media packet to another:" for line 129, "To convert one color space of an image to another:" for here would be better.

Reply:
@mijin85cho,

Thank you. I too agree with your point:
If we are going to use "To convert one color space of a media packet to another:" for line 129, "To convert one color space of an image to another:" for here would be better.